events/hfe_events_leaders.txt

```diff
@@ -1378,40 +1362,40 @@ country_event = {
 				}
 				# case CTD
 				# form pirate nation
-				# 10 = {
-				# 	modifier = {
-				# 		factor = 0
-				# 		hfe_trigger_pirate_enabled = no
-				# 	}
-				# 	modifier = {
-				# 		factor = 0
-				# 		root = {
-				# 			has_country_flag = hfe_pirate_spawned
-				# 		}
-				# 	}
-				# 	modifier = {
-				# 		factor = 0.5
-				# 		root = {
-				# 			is_pacifist = yes
-				# 		}
-				# 	}
-				# 	modifier = {
-				# 		factor = 2
-				# 		root = {
-				# 			is_militarist = yes
-				# 		}
-				# 	}
-				# 	root = {
-				# 		hfe_effect_pirate_init = yes
-				# 	}
-				# 	event_target:hfe_pirate_clan = {
-				# 		set_graphical_culture = root
-				# 		set_country_flag = hfe_leaders_refuge@root
-				# 		assign_leader = prev
-				# 		#set_name = "[root.species.GetName] Remnants"
-				# 		hfe_effect_transfer_resources_from_root_to_this = yes
-				# 	}
-				# }
+				10 = {
```

Comment:
It says it case CTD

Reply:
Fixed

Reply:
Why it still shows one line changed?

Reply:
Which line?

Reply:
Please refresh the screen. I definitely fixed this.